renderdoc/driver/gl/wrappers/gl_emulated.cpp

```diff
@@ -309,16 +309,25 @@ void APIENTRY _glGetNamedBufferParameterivEXT(GLuint buffer, GLenum pname, GLint
   internalGL->glGetBufferParameteriv(eGL_COPY_READ_BUFFER, pname, params);
 }
 
+void APIENTRY _glGetBufferSubData(GLenum target, GLintptr offset, GLsizeiptr size, void *data)
+{
+  void *bufData = internalGL->glMapBufferRange(target, offset, size, eGL_MAP_READ_BIT);
+  memcpy(data, bufData, (size_t)size);
```

Comment:
Maybe worth adding an `if(bufData)` here with an error print just to be safe? We can assume that `data` is valid but maybe the map will fail.